lib/game.rb

```diff
@@ -0,0 +1,94 @@
+class Game
+  attr_accessor :board, :player_1, :player_2  #provides access to the board, player_1, player_2
+
+  def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board=Board.new ) #Game initialize defaults to two human players, X and O, with an empty board
+    @board = board                #Game initialize accepts 2 players and a board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  WIN_COMBINATIONS = [      #defines a constant WIN_COMBINATIONS with arrays for each win combination
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [6,4,2],
+    [0,4,8]
+  ]
+
+  def over? #returns true for a draw or a won game
+    won? || draw?
+  end
+
+  def won?  #returns false for a draw
```

Comment:
Good job using `.detect` but here you can abstract this to return the winning combination or `nil` if none is found.

```ruby
  def won?
    WIN_COMBINATIONS.detect do |combo|
      @board.cells[combo[0]] == @board.cells[combo[1]] &&
      @board.cells[combo[0]] == @board.cells[combo[2]] &&
      @board.taken?(combo[0] + 1)
    end
  end
```